onepassword/cli.py

```diff
@@ -34,13 +35,19 @@ def run(self):
         )
 
         if item is not None:
-            self.stdout.write("%s\n" % item.password)
+            self.produce(item)
         else:
             self.stderr.write("1pass: Could not find an item named '%s'\n" % (
                 self.arguments.item,
             ))
             sys.exit(os.EX_DATAERR)
 
+    def produce(self, item):
+        if self.arguments.copy:
+            pyperclip.copy(item.password)
```

Comment:
I'm on the fence about whether we should notify the user that the password was copied to the clipboard. In theory, this should always either copy or print the password, or print that the item couldn't be found. No change necessary here, but I wanted to capture the idea for posterity.

Reply:
I've added it already with a bunch more functionality I needed. Will send that pr soon.